rocky/reports/views.py

```diff
@@ -58,7 +63,17 @@ def build_breadcrumbs(self):
         return breadcrumbs[: self.current_step]
 
 
-class BaseReportView(ReportBreadcrumbs, OctopoesView):
+class ReportFilterMixin:
```

Comment:
I don't understand why this is a mixin and a method call instead of just adding the forms in `get_context_data`.